proposals/utils/pdf_diff_logic.py

```diff
@@ -1029,15 +1051,17 @@ def create_context_pdf(context, model):
                     sections.append(StudyOverviewSection(study))
                     sections.append(InformedConsentFormsSection(study.documents))
 
-                extra_documents = get_extra_documents(model)
+                sections.append(KnowledgeSecuritySection(proposal))
```

Comment:
When generating a PDF for a proposal with sessions, line 1050 causes an error. I suspect this is just a mismatched indent and that the line just needs one more indentation level.

This doesn't seem to be related to this PR in particular, so I couldn't select the specific line in this review. But if you could quickly fix it here that would be great.